docset/winserver2022-ps/windowsupdate/Get-WindowsUpdateLog.md

```diff
@@ -11,28 +11,37 @@ title: Get-WindowsUpdateLog
 # Get-WindowsUpdateLog
 
 ## SYNOPSIS
+
 Merges Windows Update .etl files into a single log file.
 
 ## SYNTAX
 
 ```
-Get-WindowsUpdateLog [[-ETLPath] <String[]>] [[-LogPath] <String>] [-ProcessingType <String>] [-ForceFlush]
- [-WhatIf] [-Confirm] [<CommonParameters>]
+Get-WindowsUpdateLog [[-ETLPath] <String[]>] [[-LogPath] <String>] 
+[-ProcessingType <String>] [-ForceFlush] [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The **Get-WindowsUpdateLog** cmdlet merges and converts Windows Update .etl files into a single readable WindowsUpdate.log file.
-Windows Update Agent uses Event Tracing for Windows (ETW) to generate diagnostic logs.
-Windows Update no longer directly produces a WindowsUpdate.log file.
+
+The **Get-WindowsUpdateLog** cmdlet merges and converts Windows Update .etl files into a single
```

Comment:
We should always set cmdlet names to be wrapped in backticks so they render as code. 

```suggestion
The `Get-WindowsUpdateLog` cmdlet merges and converts Windows Update .etl files into a single
```

We have some guidance around formatting in the [contributor's guide](https://learn.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide)